docs/intro/tutorial04.txt

```diff
@@ -289,7 +289,7 @@ views and use Django's generic views instead. To do so, open the
         template_name = 'polls/detail.html'
 
 
-    class ResultsView(generic.DetailView):
+    class ResultsDetailView(generic.DetailView):
```

Comment:
IMO changes in tutorials are misleading and should be reverted them, e.g.
`IndexListView` is a main (index) view not a list of indexes, `ResultsView` is a view with a question results not details of a result.

I don't see any point in changing the current names in tutorials.